openwisp_monitoring/monitoring/migrations/0021_resources_threshold_values.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.0.3 on 2020-08-17 10:18
+
+from django.db import migrations
+
+
+resources_metrics = {'memory': 95, 'disk': 80, 'cpu': 90}
+
+
+def standardize_alertsettings_threshold(apps, schema_editor):
+    AlertSettings = apps.get_model('monitoring', 'AlertSettings')
+    for alert_settings in AlertSettings.objects.all():
+        config_name = alert_settings.metric.configuration
+        if (
+            config_name not in resources_metrics
+            or alert_settings.custom_threshold is None
+        ):
+            continue
+        alert_settings.custom_threshold *= 100
+        if alert_settings.custom_threshold == resources_metrics[config_name]:
+            alert_settings.custom_threshold = None
+        alert_settings.save()
+
+
+def reverse_alertsettings_threshold(apps, schema_editor):
+    AlertSettings = apps.get_model('monitoring', 'AlertSettings')
+    for alert_settings in AlertSettings.objects.all():
+        config_name = alert_settings.metric.configuration
+        if config_name not in resources_metrics:
+            continue
+        if alert_settings.custom_threshold is None:
+            alert_settings.custom_threshold = resources_metrics[config_name]
+        alert_settings.custom_threshold /= 100
```

Comment:
this multiplication should be done only if the current value is not an integer (or beween 0 and 1), but I think we should do this by filtering in the query rather than with an if